src/Render/Program.cs

```diff
@@ -89,13 +89,48 @@ internal static class Program {
 		}
 
 		private static string GetTitle( MarkdownDocument doc ) {
-			var inline = doc.Descendants<HeadingBlock>().Single(
-				h => h.Level == 1
-			).Inline;
-			var titleLiteral = inline.Descendants<LiteralInline>().Single();
+
+			HeadingBlock headingBlock = GetSingleTitleOrThrow( doc );
+
+			LiteralInline titleLiteral = GetUnformattedContentOrThrow( headingBlock );
+
 			return titleLiteral.Content.ToString();
 		}
 
+		private static LiteralInline GetUnformattedContentOrThrow(HeadingBlock headingBlock) {
+			var literalInlines =
+				headingBlock
+					.Inline
+					.Descendants<LiteralInline>()
+					.ToArray();
+
+			if( literalInlines.Length > 1 ) {
+				throw new ArgumentException( "The level-1 heading must be unformatted." );
+			}
+
+			var titleLiteral = literalInlines.Single();

```

Comment:
Can `literalInlines` ever be empty?

Reply:
I suspect it is technically possible, but I don't know what it would mean nor how to make it happen. Given that, I'm comfortable with it exploding.

Put another way, I could invest in making this situation better, but what's already here is an improvement.